apps/user-profile/src/app/components/header/header.component.html

```diff
@@ -7,6 +7,16 @@
   <a class="ml-3" [innerHTML]="logo" [routerLink]="disableLogo ? [] : ['/']" queryParamsHandling="merge"></a>
 
   <p [ngStyle]="{'color': textColor}" class="ml-3 mt-auto mb-auto">{{label}}</p>
-</mat-toolbar>
-
 
+  <div id="nav-menu-user-info" class="justify-content-end">
+    <perun-web-apps-header-menu
+      [activeLink]="activeLink"
+      [url]="adminGuiUrl"
+      [label]="adminLabel"
+      [user]="principal.user"
+      [logoutEnabled]="logoutEnabled"
+      [iconColor]="iconColor"
+      [textColor]="textColor">
+    </perun-web-apps-header-menu>
+  </div>
```

Comment:
```suggestion
  <perun-web-apps-header-menu
    id="nav-menu-user-info"
    class="justify-content-end"
    [activeLink]="activeLink"
    [url]="adminGuiUrl"
    [label]="adminLabel"
    [user]="principal.user"
    [logoutEnabled]="logoutEnabled"
    [iconColor]="iconColor"
    [textColor]="textColor">
  </perun-web-apps-header-menu>
```

Reply:
Div element has been removed and `id` and `class` has been moved to the element `perun-web-apps-header-menu`.